### PR TITLE
Update admqry.php

### PR DESCRIPTION
**Pull request by magicspoonouo**
bad bad github i do not like :(
anyway, this is admqry.php

### CODE REVIEW

File: adm/admqry.php [hunk 1]
```diff
@@ -1 +1,268 @@
...
+                    $check = 0;
+                    foreach ($rows as $row2) {
+                        if ($row1['stu_cd'] == $row2['staff_cd']) {
+                            echo '<tr>';
+                            // 寢別
+                            echo '<td>' . $row1['room_id'] . '</td>';
+                            // 學號
+                            echo '<td>' . $row2['staff_cd'] . '</td>';
+                            // 系所  可自行添加
+                            if ($row2['unit_parent'] == "1000")
```
Comment (Tail001) [inline adm/admqry.php:68]: You need to fetch database instead adding by yourself

File: adm/admqry.php [hunk 2]
```diff
@@ -1 +1,268 @@
...
+                        if ($row1['unit_parent'] == "1000")
+                            echo '<td>中國文學研究所</td>';
+                        else if ($row1['unit_parent'] == "2000")
+                            echo '<td>數學研究所</td>';
+                        else if ($row1['unit_parent'] == "3000")
+                            echo '<td>資訊工程研究所</td>';
+                        else if ($row1['unit_parent'] == "4000")
+                            echo '<td>心理研究所</td>';
+                        else if ($row1['unit_parent'] == "5000")
+                            echo '<td>法律研究所</td>';
```
Comment (Tail001) [inline adm/admqry.php:144]: Same as here, DB: "sl8gdm_dep" can check the depcode

File: adm/admqry.php [hunk 3]
```diff
@@ -1 +1,268 @@
...
+                            if ($row2['unit_parent'] == "1000")
+                                echo '<td>中國文學研究所</td>';
+                            else if ($row2['unit_parent'] == "2000")
+                                echo '<td>數學研究所</td>';
+                            else if ($row2['unit_parent'] == "3000")
+                                echo '<td>資訊工程研究所</td>';
+                            else if ($row2['unit_parent'] == "4000")
+                                echo '<td>心理研究所</td>';
+                            else if ($row2['unit_parent'] == "5000")
+                                echo '<td>法律研究所</td>';
```
Comment (Tail001) [inline adm/admqry.php:77]: You need to fetch database instead adding by yourself

File: adm/admqry.php [hunk 4]
```diff
@@ -1 +1,268 @@
...
+                        else if ($row1['unit_parent'] == "2000")
+                            echo '<td>數學研究所</td>';
+                        else if ($row1['unit_parent'] == "3000")
+                            echo '<td>資訊工程研究所</td>';
+                        else if ($row1['unit_parent'] == "4000")
+                            echo '<td>心理研究所</td>';
+                        else if ($row1['unit_parent'] == "5000")
+                            echo '<td>法律研究所</td>';
+                        // 學號
+                        echo '<td>' . $row1['staff_cd'] . '</td>';
```
Comment (Tail001) [inline adm/admqry.php:217]: fetch DB

**Comment by magicspoonouo** [comment]
getting another PR

**Pull request closed without merging**